cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -121,6 +142,15 @@ func (s *Server) loggedRun(tr trace.Trace, cmd *exec.Cmd) error {
 	return nil
 }
 
+func codeHostFromName(repoName string) string {
```

Comment:
@uwedeportivo you should validate the first part is a codehost name (eg contains `.` in it). repoName's don't need to contain dot and some installations don't include them (via the repo name patterns). So this change will lead to as many promeutheus metrics as repos => a problem. I'll send out a commit to fix this. Has this already been included in a release?

Reply:
Pushed to master this fix. https://github.com/sourcegraph/zoekt/commit/7b13614a083e6ec8cf03602e75c7fa409a342d37

Can you ensure it is included in any potential patch releases you are doing which include this change?

Reply:
Looks like this wasn't released, will include latest zoekt revision in 3.17.1: https://github.com/sourcegraph/sourcegraph/issues/11642